managed/ui/src/components/config/Security/KeyManagementConfigurationContainer.js

```diff
@@ -41,6 +41,13 @@ const mapDispatchToProps = (dispatch) => {
     setKMSConfig: (provider, body) => {
       return dispatch(createKMSProviderConfig(provider, body))
         .then?.((response) => {
+          if (response.error) {
+            const errorMessage =
+              response.payload?.response?.data?.error || response.payload.message;
+            toast.error(errorMessage);
+          } else {
+            toast.success('Successfully added the configuration');
```

Comment:
Success message should also come from backend , instead of static message .

Reply:
Sure, I will check with @jitendra-12113  on this.

Reply:
As per discussion with @jitendra-12113, W need to check with Sashank on this, For now `response.payload.message` is for handling generic exception.